demo/dialog-basic-demos.html

```diff
@@ -222,6 +222,40 @@ <h3>Custom Overlay Theme</h3>
       </template>
     </vaadin-demo-snippet>
 
+    <h3>Modeless Draggable Resizable Dialogs</h3>
+    <vaadin-demo-snippet id="dialog-basic-demos-modeless-draggable-resizable-dialog" when-defined="vaadin-dialog">
+      <template preserve-content>
+        <vaadin-dialog modeless draggable resizable></vaadin-dialog>
+        <vaadin-dialog modeless draggable resizable></vaadin-dialog>
+        <vaadin-button>Toggle dialog 1</vaadin-button>
+        <vaadin-button>Toggle dialog 2</vaadin-button>
+        <script>
+          window.addDemoReadyListener('#dialog-basic-demos-modeless-draggable-resizable-dialog', function(document) {
+            const dialogs = document.querySelectorAll('vaadin-dialog');
+            for (let i = 0; i < dialogs.length; i++) {
+              const dialog = dialogs[i];
+              const dialogIndex = i + 1;
+              dialog.renderer = function(root, dialog) {
+                const closeButton = window.document.createElement('vaadin-button');
+                closeButton.setAttribute('theme', 'primary');
+                closeButton.textContent = 'Close';
+                closeButton.style.display = 'inline-block';
+                closeButton.style.marginLeft = '1em';
+                closeButton.addEventListener('click', function() {
+                  dialog.opened = false;
+                });
+                root.textContent = 'Dialog ' + dialogIndex + ': Drag me around and resize me!';
```

Comment:
This explainer is a bit confusing for mobile device users. Dragging works on desktop only, so they might feel like the feature is broken.

Reply:
Should I maybe add a note above the example saying "Note the resizable and draggable features only work on desktop" ?